README.md

```diff
@@ -20,8 +20,26 @@ Releases
 
 Our [change log][changelog] has release history.
 
+Using plugin application:
 ```groovy
-implementation 'app.cash.paparazzi:paparazzi:0.1.0'
+buildscript {
+  repositories {
+    google()
+    mavenCentral()
```

Comment:
```suggestion
    mavenCentral()
    google()
```
super-nit, but doesn't really matter. otherwise you're sending all of your dependency information to Google!